app/javascript/components/App.js

```diff
@@ -28,9 +29,24 @@ class App extends Component {
   };
 
   createAuction = (auction) => {
-    fetch("/auction_items", { method: "POST", body: auction });
+    fetch("/auction_items", { method: "POST",headers:{'Content-Type': 'application/json'}, body: JSON.stringify({auction_item:auction}) });
   };
 
+  updateAuction = (updateAuction, id) => {
+		fetch('http://localhost:3000/auction_items/${id}', {
```

Comment:
You don't need to fetch to localhost. You can just fetch to 'auction_items/${id}'. Also, this doesn't look like string interpolation syntax.

Reply:
I could be wrong, but I think you will need a leading forward slash to indicate its a request to the same domain, ie `/auction_item/${id}`